shared/rakefiles/xk_infra.rake

```diff
@@ -38,7 +38,7 @@ task :refresh_common_infra, [:project_type] => [@gcp_creds_file] do | taskname,
   end
 end
 
-task :apply_common_infra => [@gcp_creds_file] do
+task :apply_common_infra => [:configure] do
   # Steps to initialize GCP with a minimum set of resources to allow Terraform
   # create the rest of the infrastructure.
   # These steps are the same found in this tutorial:
```

Comment:
Looks line Line 60 (to which Github won't allow me to attach a comment) still refers to `:configure_current_project`.

Reply:
Nice catch! Fixed.